db_schema/schema.sql

```diff
@@ -212,4 +212,21 @@ CREATE TABLE IF NOT EXISTS feedbackComment (
   comment TEXT NOT NULL,
   created TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
   PRIMARY KEY (account_id, episode_id, created)
-);
\ No newline at end of file
+);
+
+-- store latest update events
+-- timestamp is used to identify the update
+-- contains JSON with the update data
+DROP TABLE IF EXISTS updates;
+CREATE TABLE updates (
+  account_id INTEGER NOT NULL,
+  endpoint VARCHAR(64) NOT NULL,
+  timestamp TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
wouldn't recommend to use keywords as field names (timestamp TIMESTAMP), maybe just `created`?